docs/source/getting-started/examples/histograms.ipynb

```diff
@@ -197,41 +205,134 @@
       "text/plain": [
-       "<Figure size 432x288 with 1 Axes>"
+       "<Figure size 640x480 with 1 Axes>"
       ]
      },
-     "metadata": {
-      "needs_background": "light"
-     },
+     "metadata": {},
      "output_type": "display_data"
     }
    ],
    "source": [
     "preprocess = (\n",
     "    dp.t.make_split_dataframe(separator=\",\", col_names=col_names) >>\n",
     "    dp.t.make_select_column(key=\"educ\", TOA=str) >>\n",
-    "    dp.t.then_count_by(MO=dp.L1Distance[float], TV=float)\n",
+    "    dp.t.then_count_by()\n",
```

Comment:
Simple is good, but what's the error like without any change? Would a user running the old code know how to fix it?